gamla/functional.py

```diff
@@ -115,12 +109,24 @@ def inner():
 
 
 @currying.curry
-def translate_exception(func, exc1, exc2):
-    """`func` is assumed to be unary."""
+def translate_exception(func: Callable, exc1: Exception, exc2: Exception):
+    """
+    A functional try/except block: Get a function, and if it fails with `exc1`, raise `exc2`
```

Comment:
More concise:
A functional try/except block: if `func` fails with `exc1`, raise `exc2`.

(note the caps change and period at end of sentence).